filebeat/docs/modules/gsuite.asciidoc

```diff
@@ -21,18 +21,21 @@ include::../include/gs-link.asciidoc[]
 
 It is compatible with a subset of applications under the https://developers.google.com/admin-sdk/reports/v1/get-start/getting-started[Google Reports API v1]. As of today it supports:
 
-- https://developers.google.com/admin-sdk/reports/v1/appendix/activity/saml[SAML Audit Activity Events]
-- https://developers.google.com/admin-sdk/reports/v1/appendix/activity/user-accounts[User Accounts Activity Events]
-- https://developers.google.com/admin-sdk/reports/v1/appendix/activity/login[Login Audit Activity Events]
-- https://developers.google.com/admin-sdk/reports/v1/appendix/activity/admin-application-settings[Admin Audit Activity Events]
-- https://developers.google.com/admin-sdk/reports/v1/appendix/activity/drive[Drive Activity Events]
-- https://developers.google.com/admin-sdk/reports/v1/appendix/activity/groups[Groups Audit Activity Events]
+- SAML Audit Activity Events https://developers.google.com/admin-sdk/reports/v1/appendix/activity/saml[api docs] https://support.google.com/a/answer/7007375?hl=en&ref_topic=9027054[help]: View users’ successful and failed sign-ins to SAML applications.
```

Comment:
@marc-gr The content looks good, but could we use a table instead of a list? As the list of GSuite event logs grows, a table will be a much cleaner layout.